modules/age.nix

```diff
@@ -12,6 +12,8 @@ with lib; let
 
   ageBin = config.age.ageBin;
 
+  pluginPaths = concatMapStrings (x: ":" + "${x}/bin") config.age.pluginPackages;
```

Comment:
FYI there is a `lib.makeBinPath` function that will also handle accessing the proper output for you. Some packages put their binaries in the default `out` output, but some have a `bin` output, and in its current state, any plugins that have their binaries in `$bin/bin/age-plugin-asdf` might not be valid.

Reply:
Thanks, that sounds exactly like what I need :)